Library/Homebrew/bottle_version.rb

```diff
@@ -63,3 +65,25 @@ def self._parse spec
     super
   end
 end
+
+class BottlePkgVersion < PkgVersion
```

Comment:
> Why not have one class do both? The naming is kinda confusing to me.

It's because the subclassing things.

> Not sure I understand the subclassing either.

`BottleVersion` requires `Version` to work. `BottlePkgVersion` has the similar meaning to `PkgVersion`. At the same time, we can reuse the methods in the base class.